packages/core/src/tools/taskQueue.ts

```diff
@@ -0,0 +1,58 @@
+import { ONE_SECOND } from './utils/timeUtils'
+import { requestIdleCallback } from './requestIdleCallback'
+
+/**
+ * Maximum amount of time to delay the tasks. We don't want to wait too long before running tasks,
```

Comment:
<!-- Nitpicks are small, trivial, but necessary changes. Distinguishing nitpick comments significantly helps direct the reader's attention to comments requiring more involvement.  -->
**🥜 nitpick:** ‏
```suggestion
 * Maximum delay before starting to execute tasks in the queue. We don't want to wait too long before running tasks,
```